numpy/ma/core.py

```diff
@@ -3467,17 +3468,14 @@ def _get_recordmask(self):
             return _mask
         return np.all(flatten_structured_array(_mask), axis=-1)
 
-    def _set_recordmask(self):
+    @recordmask.setter
+    def recordmask(self, mask):
         """
-        Return the mask of the records.
-
-        A record is masked when all the fields are masked.
+        Set the mask of the records.
```

Comment:
Not much point putting a docstring on the setter, it's never seen by the user or sphinx.